src/main/steps/tasklistresponse/responseCaseSequence.ts

```diff
@@ -461,4 +462,9 @@ export const responseCaseSequence: Step[] = [
     showInSection: Sections.AboutRespondentCase,
     getNextStep: () => PRL_C1A_SAFETY_CONCERNS_CONCERNS_ABOUT_CHILD,
   },
+  {
```

Comment:
I do not think this is any longer required. Please verify.

Reply:
Removed as not required anymore